src/utils/helpers.ts

```diff
@@ -100,12 +70,34 @@ export function formatDate(date: Date | string, includeTime: boolean): string {
   return DateTime.fromJSDate(jsDate, { zone: 'utc' }).toFormat(format);
 }
 
-export function getSpecficCvssScore(scores: any[], issuer: string, version: string) {
-  return scores.find(
-    score => score.issuer === issuer && score.cvss_version === version,
+export function isScoreRhIssuedCvss3(score: ZodAffectCVSSType) {
```

Comment:
score is part of the cvss schema, since the function is expecting a cvss as argument, I would prefer to change the name, maybe this ?
```suggestion
export function isCVSS3issuedByRH(score: ZodAffectCVSSType) {
```